modules/pel/peltool/peltool.py

```diff
@@ -267,6 +319,8 @@ def main():
                         help='Used with -d, only look for files with this extension (e.g. ".pel")')
     parser.add_argument('-x', '--delete', dest='delete', action='store_true',
                         help='Delete original file after parsing')
+    parser.add_argument('-l', '--list', nargs='?', const="/var/lib/phosphor-logging/extensions/pels/logs/", 
+                        default=None, help='Get summary of pels present in the input directory')
```

Comment:
Let's keep the help the same as with the other peltool:  List PELs

Reply:
Resolved!